src/components/visualizations/chart/highcharts/test/getZeroAlignConfiguration.spec.ts

```diff
@@ -205,6 +209,171 @@ describe("getZeroAlignConfiguration", () => {
         });
     });
 
+    describe("getMinMaxInfo with some null data", () => {
+        const SERIES: ISeriesItem[] = [
+            {
+                yAxis: 0,
+                data: [null, null].map((num: number) => ({ y: num })),
+            },
+            {
+                yAxis: 0,
+                data: [null, null].map((num: number) => ({ y: num })),
+            },
+            {
+                yAxis: 1,
+                data: [250, null].map((num: number) => ({ y: num })),
+            },
+            {
+                yAxis: 1,
+                data: [250, 50].map((num: number) => ({ y: num })),
+            },
+        ];
+
+        it("should return min/max with non-stack chart", () => {
+            const config = {
+                yAxis: [{}, {}],
+                series: SERIES,
+            };
+            const result = getMinMaxInfo(config, null, VisualizationTypes.COLUMN);
+            expect(result).toEqual([
+                {
+                    id: 0,
+                    min: 0,
+                    max: 0,
+                    isSetMin: false,
+                    isSetMax: false,
+                },
+                {
+                    id: 1,
+                    min: 0,
+                    max: 250,
+                    isSetMin: false,
+                    isSetMax: false,
+                },
+            ]);
+        });
+
+        it("should return min/max in total with normal stack chart", () => {
+            const config = {
+                yAxis: [{}, {}],
+                series: SERIES,
+            };
+            const result = getMinMaxInfo(config, NORMAL_STACK, VisualizationTypes.COLUMN);
+            expect(result).toEqual([
+                {
+                    id: 0,
+                    min: 0,
+                    max: 0,
+                    isSetMin: false,
+                    isSetMax: false,
+                },
+                {
+                    id: 1,
+                    min: 0,
+                    max: 500,
+                    isSetMin: false,
+                    isSetMax: false,
+                },
+            ]);
+        });
+
+        it("should return min/max in total with percent stack chart", () => {
```

Comment:
Should we use data driven test for this 3 tests? (line 232, 256 and 280)

Reply:
done